core-ajax.html

```diff
@@ -34,8 +34,7 @@
 -->
 
 <link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="core-request.html">
-<dom-element id="core-ajax"></dom-element>
+<link rel="import" href="./core-request.html">
```

Comment:
I haven't seen it elsewhere, so I'd remove it unless there's a technical reason for it. Going by repo consensus on that.